### PR TITLE
Parsing ICL with image data

### PR DESCRIPTION
**Pull request by jenakan**
Line length changes to support ASCII encoded ICLs with type(52) ImageData. Implemented a secondary format that will scan for lines based on control bytes defined by spec rather than CRLF line breaks.

Not sure if this counts as a format as mentioned in https://github.com/moov-io/imagecashletter/issues/111.

### CODE REVIEW

**Comment by adamdecaf** [comment]
Awesome, thanks @jenakan!

File: file.go [hunk 1]
```diff
@@ -77,6 +77,16 @@ func (e *FileError) Error() string {
 	return fmt.Sprintf("%s %s", e.FieldName, e.Msg)
 }
 
+// Format standard of X9.37 specification used to parse file
+type Format uint32
+
+const (
+	// Discover format
+	Discover Format = iota
```
Comment (adamdecaf) [inline file.go:85]: What if we renamed this to `Detect`? I'm worried Discover (the company) might have an X9 format we run into one day...

**Review by adamdecaf — APPROVED** [review]

File: file.go (hunk 1 above)
Comment (jenakan) [reply file.go:85]: @adamdecaf Thinking of dropping the Format enum. It's too restrictive.Unfortunately FRB adoption of DTSU requires EBCDIC encoding. I'm making some changes to allow reading/writing with configurable options. It should allow more flexibility for others.

Comment (adamdecaf) [reply file.go:85]: Okay. We'll write up a matrix to explain the known combinations.

**Pull request merged**